arch/src/aarch64/fdt.rs

```diff
@@ -159,6 +280,54 @@ fn create_cpu_nodes(
 
     let num_cpus = vcpu_mpidr.len();
 
+    //Add cache info
```

Comment:
nit: Give some whitespace after the comment start.

Reply:
Thanks,  I will modify it soon.